src/components/table/AposTable.stories.js

```diff
@@ -0,0 +1,34 @@
+import { withKnobs, boolean } from '@storybook/addon-knobs';
+
+import AposTable from './AposTable.vue';
+
+import data from './data.js';
+
+export default {
+  title: 'Table',
+  decorators: [withKnobs]
+};
+
+export const Table = () => ({
+  components: {
+    AposTable
+  },
+  data() {
+    return {
+      data,
+      selectable: false,
+      tree: false
+    };
+  },
+  props: {
+    selectable: {
+      default: boolean('Selectable', false)
+    }
+  },
+  template: `
+    <AposTable
+      :data="data"
```

Comment:
generally we break up the data a bit so we can provide defaults and more specific Types, `headers` and `rows` in this case